### PR TITLE
Changes to company identity creation

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about version 1 flow.

## Changes

Update links in topic.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
How do we want to handle a note about version 1 flow vs. version 0 flow? Do they have to use version 1? Recommended?

**Comment by abizjak** [comment]
So these changes are intended to be merged after we release the updated tools with the new flows, right?

**Comment by dg-concordium** [comment]
> So these changes are intended to be merged after we release the updated tools with the new flows, right?

Correct. I never release anything until you've released the tools.

**Review by abizjak — APPROVED** [review]
Looks good to me.

File: source/mainnet/net/guides/company-identities.rst [hunk 1]
```diff
@@ -7,21 +7,23 @@ Company identity creation
...
+The tabs below describe how to create a company identity. Note that the process differs for testnet and mainnet.
 
 .. tabs::
 
    .. tab:: Mainnet
 
       #. Download the tools for your platform.
 
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`__
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:20]:
```suggestion
         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities-v2.tar.gz>`__
```

File: source/mainnet/net/guides/company-identities.rst [hunk 2]
```diff
@@ -7,21 +7,23 @@ Company identity creation
...
             - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:21]:
```suggestion
            - SHA256 checksum of the download: ``fd3620f3f3e2e9540b262ae68b8273c59816fbaa12d495629b07555c65bab4a2``
```

File: source/mainnet/net/guides/company-identities.rst [hunk 3]
```diff
@@ -7,21 +7,23 @@ Company identity creation
...
 
-         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`_
+         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`__
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:23]:
```suggestion
         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities-v2.zip>`__
```

File: source/mainnet/net/guides/company-identities.rst [hunk 4]
```diff
@@ -7,21 +7,23 @@ Company identity creation
...
             - SHA256 checksum of the download: ``4a13eec29b6c7fc8214555c6b13e431c20df449bfd11e1a2f26b6a6e91a03957``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:24]: TODO, the link does not yet work

File: source/mainnet/net/guides/company-identities.rst [hunk 5]
```diff
@@ -7,21 +7,23 @@ Company identity creation
...
 
-         - `Tools for MacOS <https://distribution.concordium.software/tools/macos/signed/enterprise-identities.zip>`_
+         - `Tools for MacOS <https://distribution.concordium.software/tools/macos/signed/enterprise-identities.zip>`__
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:26]:
```suggestion
         - `Tools for MacOS <https://distribution.concordium.software/tools/macos/signed/enterprise-identities-v2.zip>`__
```

File: source/mainnet/net/guides/company-identities.rst [hunk 6]
```diff
@@ -7,21 +7,23 @@ Company identity creation
...
             - SHA256 checksum of the download: ``242a0ff19f84c91ec0c1a3c7696718eee61f144a73a3700d8969f3531384ad6e``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:27]:
```suggestion
            - SHA256 checksum of the download: ``1bb44659a66dcd8b60fed5f43d53d9575e8e35710ac785a87a9c161c51b1acc7``
```

File: source/mainnet/net/guides/company-identities.rst [hunk 7]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`__
             - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
 
-         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`_
+         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`__
             - SHA256 checksum of the download: ``4a13eec29b6c7fc8214555c6b13e431c20df449bfd11e1a2f26b6a6e91a03957``
 
-         - `Tools for MacOS <https://distribution.concordium.software/tools/macos/signed/enterprise-identities.zip>`_
+         - `Tools for MacOS <https://distribution.concordium.software/tools/macos/signed/enterprise-identities.zip>`__
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:68]:
```suggestion
         - `Tools for MacOS <https://distribution.concordium.software/tools/macos/signed/enterprise-identities-v2.zip>`__
```

File: source/mainnet/net/guides/company-identities.rst [hunk 8]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
             - SHA256 checksum of the download: ``242a0ff19f84c91ec0c1a3c7696718eee61f144a73a3700d8969f3531384ad6e``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:69]:
```suggestion
            - SHA256 checksum of the download: ``1bb44659a66dcd8b60fed5f43d53d9575e8e35710ac785a87a9c161c51b1acc7``
```

File: source/mainnet/net/guides/company-identities.rst [hunk 9]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
 
       #. Download the tools for your platform.
 
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`__
             - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
 
-         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`_
+         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`__
             - SHA256 checksum of the download: ``4a13eec29b6c7fc8214555c6b13e431c20df449bfd11e1a2f26b6a6e91a03957``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:66]: TODO

File: source/mainnet/net/guides/company-identities.rst [hunk 10]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
+
+      If you experience issues with steps 1, 2, 3, 4, 6 or 7, please contact Concordium’s technical support via support@concordium.software. If you experience issues with step 5, identity verification, please contact Notabene via ania@notabene.id.
 
 
    .. tab:: Testnet
 
       #. Download the tools for your platform.
 
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`__
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:62]:
```suggestion
         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities-v2.tar.gz>`__
```

File: source/mainnet/net/guides/company-identities.rst [hunk 11]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
             - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:63]:
```suggestion
            - SHA256 checksum of the download: ``fd3620f3f3e2e9540b262ae68b8273c59816fbaa12d495629b07555c65bab4a2``
```

File: source/mainnet/net/guides/company-identities.rst [hunk 12]
```diff
@@ -7,22 +7,24 @@ Company identity creation
...
       #. Download the tools for your platform.
 
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
-            - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities-v2.tar.gz>`__
+            - SHA256 checksum of the download: ``fd3620f3f3e2e9540b262ae68b8273c59816fbaa12d495629b07555c65bab4a2``
 
-         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`_
+         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities-v2.zip>`__
             - SHA256 checksum of the download: ``4a13eec29b6c7fc8214555c6b13e431c20df449bfd11e1a2f26b6a6e91a03957``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:24]:
```suggestion
            - SHA256 checksum of the download: ``38433e51efa95121ee4e25a15552dd02905193e3de5d3976e4b067bd9cb46096``
```

File: source/mainnet/net/guides/company-identities.rst [hunk 13]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
       #. Download the tools for your platform.
 
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
-            - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities-v2.tar.gz>`__
+            - SHA256 checksum of the download: ``fd3620f3f3e2e9540b262ae68b8273c59816fbaa12d495629b07555c65bab4a2``
 
-         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`_
+         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`__
             - SHA256 checksum of the download: ``4a13eec29b6c7fc8214555c6b13e431c20df449bfd11e1a2f26b6a6e91a03957``
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:66]:
```suggestion
            - SHA256 checksum of the download: ``38433e51efa95121ee4e25a15552dd02905193e3de5d3976e4b067bd9cb46096``
```

File: source/mainnet/net/guides/company-identities.rst (hunk 9 above)
Comment (abizjak) [reply source/mainnet/net/guides/company-identities.rst:66]: Link now works.

File: source/mainnet/net/guides/company-identities.rst (hunk 4 above)
Comment (abizjak) [reply source/mainnet/net/guides/company-identities.rst:24]: Link now works.

File: source/mainnet/net/guides/company-identities.rst [hunk 14]
```diff
@@ -34,40 +36,39 @@ You can't use the Desktop Wallet, |mw-gen2|, or |mw-gen1| to create a company id
...
 
       #. Download the tools for your platform.
 
-         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities.tar.gz>`_
-            - SHA256 checksum of the download: ``b9981c542f46e92dd05e8b3e9bf46684e9de364bd331cd6fa8db98ed99b4df84``
+         - `Tools for Linux <https://distribution.concordium.software/tools/linux/enterprise-identities-v2.tar.gz>`__
+            - SHA256 checksum of the download: ``fd3620f3f3e2e9540b262ae68b8273c59816fbaa12d495629b07555c65bab4a2``
 
-         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`_
+         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities.zip>`__
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:65]:
```suggestion
         - `Tools for Windows <https://distribution.concordium.software/tools/windows/signed/enterprise-identities-v2.zip>`__
```

**Review by abizjak — APPROVED** [review]
I added the new hashes, and the windows link now works.

**Pull request merged**